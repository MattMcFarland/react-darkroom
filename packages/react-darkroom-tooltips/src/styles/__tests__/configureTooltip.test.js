import {
  configureTooltip,
} from '../index';

const { describe, test, expect } = global;

describe('styles/configureTooltip', () => {
  test('configures tooltips for future use', () => {
    const options = {
      position: 'test',
      radius: 'test',
      color: 'test',
      arrowSize: 'test',
    };
    expect(configureTooltip(options)).toMatchSnapshot();
  });
});
