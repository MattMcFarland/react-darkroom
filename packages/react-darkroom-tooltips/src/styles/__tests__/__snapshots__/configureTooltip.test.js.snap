// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`styles/configureTooltip configures tooltips for future use 1`] = `
Object {
  "&:after": Object {
    "backgroundColor": "test",
    "borderRadius": "test",
    "boxShadow": "1px 1px 3px rgba(0, 0, 0, .2)",
    "color": "white",
    "content": "attr(data-label)",
    "padding": "5px 10px",
    "textShadow": "0 -1px 0 rgba(0, 0, 0, .2)",
    "top": "135%",
    "whiteSpace": "nowrap",
  },
  "&:before": Object {
    "border": "test solid transparent",
    "borderBottomColor": "test",
    "bottom": "-35%",
    "content": "\\"\\"",
    "zIndex": 1000001,
  },
  "&:before, &:after": Object {
    "fontSize": 12,
    "left": "50%",
    "lineHeight": "18px",
    "opacity": 0,
    "pointerEvents": "none",
    "position": "absolute",
    "transform": "translateX(-50%)",
    "transition": "250ms ease-in-out",
    "visibility": "hidden",
    "zIndex": 1000000,
  },
  "&:hover, &:focus": Object {
    "&:after": Object {
      "top": "115%",
    },
    "&:before": Object {
      "bottom": "-15%",
    },
    "&:before, &:after": Object {
      "opacity": 1,
      "visibility": "visible",
    },
  },
  "&[disabled]": Object {
    "&:before, &:after": Object {
      "display": "none",
    },
  },
  "display": "inline-block",
  "lineHeight": 1.125,
  "position": "relative",
}
`;
